gp-read-only/gpro-disable-for-admins.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Gravity Perks // GP Read Only // Disable Read Only for Admins
+ * https://gravitywiz.com/documentation/gravity-forms-read-only/
+ *
+ * This snippet allows you to disbale the Read Only property on fields for Admin users.
```

Comment:
Typo: disbale → disable. 

Not absolutely necessary here since the _why_ is probably easy for most folks to grasp but if I were to add one it'd be something like:
> Disable the Read Only property on fields for Admin users. This is useful when wanting to allow administrative users to edit fields that non-admins should not be able to edit.

Reply:
Thanks for this. I've gotten a better understanding of how to compose the description.